src/primitives/transaction.h

```diff
@@ -61,13 +61,39 @@ class CTxIn
     CScript scriptSig;
     uint32_t nSequence;
 
+    /* Setting nSequence to this value for every input in a transaction
+     * disables nLockTime. */
+    static const uint32_t SEQUENCE_FINAL = 0xffffffff;
```

Comment:
The comments here are slightly confusing to me because of missing context.  The above is a consensus rule for all transactions, while the ones below only apply as policy (for now) for `tx.nVersion >= 2` transactions.  I think we should expand on the comments to make this clearer, perhaps with a reference to BIP68?